packages/back-end/src/models/SavedGroupModel.ts

```diff
@@ -0,0 +1,79 @@
+import mongoose from "mongoose";
+import { SavedGroupInterface } from "../../types/saved-group";
+import uniqid from "uniqid";
+
+const savedGroupSchema = new mongoose.Schema({
+  id: {
+    type: String,
+    unique: true,
+  },
+  orgId: {
+    type: String,
+    index: true,
+  },
+  groupName: String,
+  owner: String,
+  dateCreated: Date,
+  dateUpdated: Date,
+  values: [String],
+  attributeKey: String,
+});
+
+type SavedGroupDocument = mongoose.Document & SavedGroupInterface;
+
+const SavedGroupModel = mongoose.model<SavedGroupDocument>(
+  "savedGroup",
+  savedGroupSchema
+);
+
+function parseSaveGroupString(list: string) {
+  const listArr = list.split(",");
+
+  const savedGroup = listArr.map((i: string) => {
+    return i.trim();
+  });
+
+  return [
+    ...new Set(savedGroup.filter((value) => value !== "," && value !== "")),
+  ];
+}
+
+type SavedGroupEditableProps = Omit<
+  SavedGroupInterface,
+  "dateCreated" | "dateUpdated" | "id" | "values"
+>;
+
+export async function createSavedGroup(
+  values: string,
+  group: SavedGroupEditableProps
+): Promise<SavedGroupInterface> {
+  const newGroup = await SavedGroupModel.create({
+    ...group,
+    values: parseSaveGroupString(values),
+    id: uniqid("grp_"),
+    dateCreated: new Date(),
+    dateUpdated: new Date(),
+  });
+  return newGroup.toJSON();
+}
```

Comment:
I would probably have this function just take a single object. The pattern you're using is nice when there's one required argument and a bunch of optional ones. But in this case, most (or all of them) are required, so there's not a good reason to pick just one out to be a standalone argument.